common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -806,8 +798,9 @@ function (HTML5Video, Resizer) {
 
     function updatePlayTime(time, skip_seek) {
         var videoPlayer = this.videoPlayer,
-            duration = this.videoPlayer.duration(),
+            duration = (this.config.endTime !== null) ? this.config.endTime : this.videoPlayer.duration(),
```

Comment:
See my other note about `null` vs. `undefined` for the `endTime` value. It looks like sometimes we're using/checking one, sometimes the other; sometimes using `!==`, others using `!=`.

Whatever we decide, I'd like the convention to be consistent. My initial preference is to just use `undefined` if no value has been specified.